supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -1,4 +1,15 @@
 // OCP: Added "In Red Hat OpenShift, namespace is"
+
+[[n-way-replication]]
+==== image:images/yes.png[yes] n-way replication (noun)
+*Description*: Generically, refers to data being copied to multiple storage devices so that _n_ copies are available in the storage cluster as a whole. This term is generally only used in Red Hat Gluster Storage.
```

Comment:
```suggestion
*Description*: The term _n-way replication_ is a generic phrase that refers to data being copied to multiple storage devices so that _n_ copies are available in the storage cluster as a whole. This phrase is only used in Red Hat Gluster Storage.
```